src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -16,17 +16,38 @@ const CHART_KEY = '__CHART_KEY__';
  *
  * @param {Array} data Data returned from Snuba
  * @param {Object} query Query state corresponding to data
+ * @param {Object} [options] Options object
+ * @param {Boolean} [options.includePercentages] (default: false) Include aggregation's percentages
  * @returns {Array}
  */
-export function getChartData(data, query) {
+export function getChartData(data, query, options = {}) {
   const {fields} = query;
 
+  const totalsBySeries = new Map();
+
+  if (options.includePercentages) {
+    query.aggregations.forEach(aggregation => {
+      totalsBySeries.set(
+        aggregation[2],
+        data.reduce((acc, res) => {
+          acc += res[aggregation[2]];
+          return acc;
+        }, 0)
+      );
+    });
+  }
+
   return query.aggregations.map(aggregation => {
+    const total = options.includePercentages && totalsBySeries.get(aggregation[2]);
     return {
       seriesName: aggregation[2],
       data: data.map(res => {
         return {
           value: res[aggregation[2]],
+          percentage:
```

Comment:
nit: We shouldn't set `percentage` at all if we're `includePercentages` is false